src/Bindings/PluginManager.h

```diff
@@ -444,6 +444,12 @@ class cPluginManager
 
 	/** Returns the folders that are specified in the settings ini to load plugins from. */
 	AStringVector GetFoldersToLoad(cSettingsRepositoryInterface & a_Settings);
+
+	/** Calls a_HookFunction on each plugin registered to the hook HookName.
+	Returns false if the action is to continue or true if the plugin wants to abort.
+	Accessible only from within PluginManager.cpp */
+	template <class HookFunction>
+	bool GenericCallHook(PluginHook a_HookName, HookFunction a_HookFunction);
```

Comment:
could be a `const` function 😼 .

Reply:
It could be but the question is should it be?
It can arbitrarily modify the state of plugins and so isn't really `const`. It does depend on whether you think the plugins count as part of the manager though.